README.md

```diff
@@ -85,6 +85,53 @@ it will be downloaded and installed in the source tree.
 #### Debian-based distro:
 
     sudo apt-get install nodejs git build-essential python2.7
+    #Raspberry Pi Cjdns Install
+
+##### Raspberry Pi
```

Comment:
The script in the Toronto repo is quite specific and does not only install cjdns. I suggest extracting part of that script, put them in contrib here, and reference that instead.